NeoML/include/NeoML/Dnn/Layers/MatrixMultiplicationLayer.h

```diff
@@ -41,6 +45,14 @@ class NEOML_API CMatrixMultiplicationLayer : public CBaseLayer {
 	void RunOnce() override;
 	void BackwardOnce() override;
 	int BlobsForBackward() const override { return TInputBlobs; }
+
+private:
+	enum TSMMD { TSMMD_Forward, TSMMD_Backward, TSMMD_SecondBackward, /*...*/ TSMMD_Count_ };
+	CSmallMatricesMultiplyDesc* smallMatricesMulDescs[TSMMD_Count_]{}; // execution descriptor
```

Comment:
`CPointerArray` and destructor won't be needed

Reply:
fixed